### PR TITLE
Building

### PR DESCRIPTION
**Pull request by obed300**
In this project, I have buid an app called the stock listening app which is a web application that displays the list of companies on the NASDAQ stock exchange and allow users to search for companies and sort the list by different criteria, such as name and symbol. Also, when the user clicks (or taps) on a category item, the application navigates to the details page

  ## live video link
  https://www.loom.com/share/1cba6f4653024c0e9fcb23e7379431cb

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* body {
```
Comment (MahmoudBakr23) [inline src/index.css]: Required
- Please remove any unused code for a cleaner code base and more optimized running app

File: README.old.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+<a name="readme-top"></a>
```
Comment (MahmoudBakr23) [inline README.old.md]: Required
- Please remove the old from the readme file name in order to appear in the repository home page

File: README.old.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+
+- **[Display a list of companies on NASDAQ ]**
+- **[Allow users to search for companies by name or symbol  ]**
+- **[When a category item is clicked, it send the user to the detail page where the user can see in more detail about the company ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[live Demo](https://stocklistening.netlify.app/)
```
Comment (MahmoudBakr23) [inline README.old.md]: Required
- Kindly fix the live demo link because it's not working

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @obed300 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Great work with your components
- Nice job using react hooks
- Well done using redux  to configure the store and create your reducers' slices
- Great file structure
- Good work with the filtering method
- Well written tests
- Nice design 
- Descriptive PR and README file
- Correct Linters
- Good presentation 

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alexander16108** [comment]
Hi @obed300 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that failing linter errors
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by sumairq — APPROVED** [review]
### Hi @obed300 , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**